lib/tasks/seed.rake

```diff
@@ -1,13 +1,13 @@
-require 'faker'
+require 'faker/user'
```

Comment:
Probably in separate change, but maybe we can add the `require_all` gem and `require_all 'faker'`?

Reply:
Does shopify have custom autoloading code for the lib directory?

Reply:
Hmm, not actually sure. We can take a look on Tuesday.